src/screens/AboutScreen.js

```diff
@@ -17,18 +17,20 @@ import {
 } from '../components';
 import { getQuery, QUERY_TYPES } from '../queries';
 import { graphqlFetchPolicy, refreshTimeFor } from '../helpers';
+import { useMatomoTrackScreenView } from '../hooks';
 
 export const AboutScreen = ({ navigation }) => {
   const [refreshTime, setRefreshTime] = useState();
   const { isConnected, isMainserverUp } = useContext(NetworkContext);
   const globalSettings = useContext(GlobalSettingsContext);
   const [refreshing, setRefreshing] = useState(false);
+  const { MATOMO_TRACKING, REFRESH_INTERVALS } = consts;
 
   useEffect(() => {
     const getRefreshTime = async () => {
```

Comment:
this should also not be redefined every time the `useEffect` is called.

the `const refresh` function below should also be in a `useCallback`

Reply:
thanks, needs to be refactored with #121